src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/lagging_vendor.phpt

```diff
@@ -33,7 +33,7 @@ require __DIR__.'/fake_vendor/acme/outdated-lib/outdated_file.php';
 
 ?>
 --EXPECTF--
-Remaining indirect deprecation notices (1)
+Indirect deprecation notices (1)
```

Comment:
Why remove the "remaining" prefix? It made the output more friendly to me, I see no reasons to drop it.

Reply:
It was my suggestion, check https://github.com/symfony/symfony/pull/35271#discussion_r367619440. Remaining implies non-remaining ones are displayed somewhere else. Please help us see how it improves friendliness using that word.

Reply:
When you're in the process of fixing deprecations - ie when you look at those reports, there are the "remaining deprecations" triggered by your code, until you fix them all. I don't read remaining as you and thus I don't see the confusion. Writing "Indirect deprecation notices (1)" feels too... direct. Remaining gives a direction to the cold fact, thus helps humans (because human don't know what to do with cold facts, but that's going off topic :) )

Reply:
Why not „Deprecations you can fix“ and „Vendor Deprecations“? 
Another wording is fine too, you get my point... hopefully 🙂

Reply:
Unless there is an issue with the current wording, reducing merge conflicts is my priority, thanks for understanding :)

Reply:
We can say the same about normal failing phpunit test cases. They are supposed/waiting to be fixed, but phpunit doesn't say "remainig failures". AFAIK nowhere else than phpunit bridge is this concept used like this. 

Also, in practice user can't fix Indirect deprecations, but they are "remaining". So there is nothing to fix for user similar as "legacy" deprecations, but unlike legacy deprecations they are for some reason marked as "remaining".

To me whole remaining concept just makes things more confusing. But since we disagree, I guess this should be extracted to separate PR/issue.

Reply:
@OskarStark also, "remaining" was also used for vendor deprecations :P

Reply:
"vendor" deprecations are not a thing
"indirect" are - and YOU are always responsible for them, all of them. Could be because YOU decided to use this package or YOU didn't contribute the fix yet, or YOU are using the package in a deprecated way.
Let's stop the wrong narative about "it them, not me".